web/client/src/library/components/graph/ModelLineage.tsx

```diff
@@ -346,7 +348,7 @@ function ModelColumnLineage(): JSX.Element {
       >
         <Panel
           position="top-right"
-          className="bg-theme !m-0 w-full"
+          className="bg-theme !m-0 w-full !z-10"
```

Comment:
without this, ran into an issue where the lineage information popover was underneath the react flow controls when I was working on my tiny laptop screen 

before
<img width="446" alt="image" src="https://github.com/TobikoData/sqlmesh/assets/81378/cdec8d1b-2eb6-46f9-bcba-e2e156991010">

after
<img width="441" alt="image" src="https://github.com/TobikoData/sqlmesh/assets/81378/af09828f-076a-4fe6-9bc5-7e498fa30b10">